python/ql/src/Security/CWE-074/TemplateInjection.qhelp

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE qhelp SYSTEM "qhelp.dtd">
+<qhelp>
+    <overview>
+        <p>
+                A template from a server templating engine such as Jinja constructed from user input can allow the user to execute arbitrary code using certain template features. It can also allow for cross-site scripting.
+        </p>
+    </overview>
+    <recommendation>
+        <p>
+            Ensure that an untrusted value is not used to directly construct a template. 
+            Jinja also provides a <code>SandboxedEnvironment</code> that prohibits access to unsafe methods and attributes, that can be used if constructing a template from user input is absolutely necessary. 
```

Comment:
```suggestion
            Jinja also provides a <code>SandboxedEnvironment</code> that prohibits access to unsafe methods and attributes. This can be used if constructing a template from user input is absolutely necessary. 
```